keyboards/alpha/alpha.h

```diff
@@ -0,0 +1,16 @@
+#ifndef KB_H
+#define KB_H
+
+#include "quantum.h"
+
+#define KEYMAP( \
```

Comment:
I believe that QMK is switching over to `LAYOUT` here, rather than `KEYMAP`.

Reply:
I don't understand what needs to be changed here and what's going wrong, could you clarify please?

Reply:
They're basically the same, but we've been moving to using primarily `LAYOUT`.  

The Layouts feature allows you to share the same keymap between multiple keyboards, as long as they're the same layout. So you could use the same keymap for the Planck, Lets Split, Nui Mini, etc. 

https://docs.qmk.fm/features/layouts